public/app/plugins/panel/nodeGraph/usePanning.ts

```diff
@@ -184,11 +184,10 @@ function inBounds(value: number, min: number | undefined, max: number | undefine
 }
 
 function getEventXY(event: Event): { x: number; y: number } {
-  if (event instanceof TouchEvent) {
+  if ((event as TouchEvent).changedTouches && event instanceof TouchEvent) {
```

Comment:
The issue here is that TouchEvent is undefined so using instanceof in Firefox and Safari will throw an exception but if it's `(event as TouchEvent).changedTouches` it will be undefined and the if check will fail so it will go to the else but will not throw an exception